ci2/ci/ci.py

```diff
@@ -75,19 +75,33 @@
     config = {}
     config['number'] = pr.number
     if pr.batch:
-        status = await pr.batch.status()
-        for j in status['jobs']:
-            if 'duration' in j and j['duration'] is not None:
-                j['duration'] = humanize.naturaldelta(datetime.timedelta(seconds=j['duration']))
-            attrs = j['attributes']
-            if 'link' in attrs:
-                attrs['link'] = attrs['link'].split(',')
-        config['batch'] = status
+        config['batch'] = await batch_status_template_context(pr.batch)
         config['artifacts'] = f'{BUCKET}/build/{pr.batch.attributes["token"]}'
+    if pr.most_recent_build is not None:
+        config['most_recent_build'] = {
+            'source_sha': pr.most_recent_build.source_sha,
```

Comment:
most_recent_build is not a thing anymore.  Grab the source, etc. from the most_recent_batch attributes.